src/topp/IDMerger.cpp

```diff
@@ -356,7 +363,9 @@ class TOPPIDMerger :
         }
         peptides.insert(peptides.end(), base_peptides.begin(),
                         base_peptides.end());
-        // merge in data from other files:
+        ///cut all other PeptideIdentifications to the first hit after sorting
+        /// and add new sequences, add the first ProteinHits corresponding to
+        /// the new sequence
```

Comment:
no but not much worse (at least regarding my understanding on what is happening in the code)